docs/_docs/Starters Academy (LDSSA)/04-Bootcamp-Structure.md

```diff
@@ -0,0 +1,83 @@
+| SLU05 | Covariance & Correlation                 | Bootcamp Day 1  |  Yes      |
+| SLU06 | Dealing with Data Problems               | Bootcamp Day 1  |  Yes      |
+| SLU07 | Regression with Linear Regression        | Bootcamp Day 1  |  Yes      |
+| SLU08 | Metrics for Regression                   | Bootcamp Day 1  |  Yes      |
+| SLU09 | Classification with Logistic Regression  | Bootcamp Day 1  |  Yes      |
+| SLU10 | Metrics for Classification               | Bootcamp Day 1  |  Yes      |
+| SLU11 | Tree-Based Models                        | Bootcamp Day 2  |  Yes      |
+| SLU12 | Feature Engineering (aka Real Wold Data) | Bootcamp Day 2  |  Yes      |
+| SLU13 | Model Selection & Overfitting            | Bootcamp Day 2  |  Yes      |
+| SLU14 | Data Sufficiency & Selection             | Bootcamp Day 2  |  Yes      |
+| SLU15 | Ethics & Fairness                        | Bootcamp Day 2  |  Yes      |
+| SLU16 | Hyperparameter Tuning                    | Bootcamp Day 2  |  Yes      |
+| SLU17 | Workflow                                 | Bootcamp Day 2  |  Yes      |
+| SLU18 | Support Vector Machines (SVM)            | Not presented   |  No       |
+| SLU19 | k-Nearest Neighbors (kNN)                | Not presented   |  No       |
+
+During the days of the bootcamp, each presented unit will follow the following schedule:
+
+- 15 min presentation
+- 5 min questions
+- 40 min of working time on the unit
+
+However, it is not expected that in the 40 min of working time you complete all of the content, 
+it is just a first contact with the content. The bootcamp days are followed by 3 weeks where
+you are expected to review the materials and complete all units by yourself.
+
+
+
+### Time commitment
+
+Aside from the bootcamp weekend, you should be prepared to spend a recommended *10 hours per week* 
+on the bootcamp units throughout of the course of the 3 weeks given to complete them.
+
+
+### Bootcamp Evaluation Rules
+
+In order to pass the bootcamp:
+
+* The student must have been present in the bootcamp days
+* All mandatory units should have been submitted and score at least 16/20 until the deadline (see [below](#Calendar))
+
+
+### Optional units
+
+There are a few optional units that we recommend the students look at, but that are not mandatory to 
+pass the bootcamp. However, they will show up on the final curriculum as either passed/failed. As other
+units a passing grade is considered for submissions that score at least 16/20 until the deadlines 
+(see [below](#Calendar))
+
+We advise you to only tackle these after you have completed all mandatory content, since the bootcamp
+is essential to proceed with the academy. 
+
+
+### Calendar
+
+| Description |  Date | 
+|-------------|-------------|
+| Bootcamp day 1 |  24th July 2021, from 8h30 AM to 7h00 PM | 
+| Bootcamp day 2 |  25th July 2021, from 8h30 AM to 7h00 PM |
+| Deadline for bootcamp units |  19th August 2021, 23h59 |
```

Comment:
should read "14th August 2021, 23h59 UTC", as per the Starters Academy Schedule

Reply:
Fixed